### DISCUSSION TITLE
Welcome to Azure-Monitor-for-SAP-solutions-preview Discussions!

### DISCUSSION THREAD

**Discussion by MightySuz** [Azure/Azure-Monitor-for-SAP-solutions-preview-#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shamilmca** [comment]
Hello Team,
We have setup the Azure Monitor for SAP Solution for HANA DB with HANA Provider.
Here the problem is that we get the alert for HANA DB unavailability even though HANA DB is up and running fine.
AMS is not able to PING to the server to get the DB status. 
Can please assist to fix it.
Alert : Fired when a SAP HANA system hasn't reported a PING alive for the last 20 mins. 
let StartTime=now(); SapHana_SystemAvailability_CL 
| where PROVIDER_INSTANCE_s == 'xxxxxxxx' 
| where EVENT_NAME_s == 'PING' |top 1 by TimeGenerated desc 
| extend AggregateValue = iff(StartTime - TimeGenerated > 20min, 1, 0) 
| summarize AggregatedValue = max(AggregateValue) by bin(TimeGenerated, 5m)
